ntpoffset.py

```diff
@@ -24,23 +24,29 @@ def warn(msg):
 class NtpOffsetConfigException(Exception):
     pass
 
-
 class NtpOffset(object):
     PLUGIN_NAME = 'ntpoffset'
 
     def __init__(self, config=None):
         self.pool = None
+        self.absolutes = False
         self.ntp_client = ntplib.NTPClient()
         if config:
             self.config(config)
 
     def config(self, conf):
-        entries = [node for node in conf.children if node.key == 'pool']
+        for node in conf.children:
+          if node.key == 'pool':
+            config_pool = node
+          elif node.key == 'absolutes':
+            self.absolutes = str(node.values[0]).lower() in ['true', 'yes', 't', '1', 'on']
```

Comment:
This expression should be moved into a named function, eg: `self.absolutes = is_truthy(node.values[0])`